osbs/utils.py

```diff
@@ -244,6 +247,9 @@ def checkout_git_repo(git_uri, git_ref, git_branch=None, max_retries=GIT_MAX_RET
                                 "branch '%s'" % (git_uri, git_branch),
                                 cause=ex, traceback=sys.exc_info()[2])
 
+        if git_branch and not depth:
+            git_depth_cmd = 'git log --oneline ...{0} | wc -l'.format(git_ref)
```

Comment:
Is git_ref being sanitized here?
Otherwise this would allow a user to execute arbitrary commands in the cluster.
Maybe we could drop the `git log --oneline` in favor of  `git rev-list --count` -1?

Reply:
Didn't know about `git rev-list --count` and will happily use that instead.